frontend/src/actions/extractors.js

```diff
@@ -0,0 +1,24 @@
+import {V2} from "../openapi";
+import {handleErrors} from "./common";
+import config from "../app.config";
+import {getHeader} from "../utils/common";
+
+export const RECEIVE_EXTRACTORS = "RECEIVE_EXTRACTORS";
+
+export function fetchExtractors(){
+	return (dispatch) => {
+		// TODO: Parameters for dates? paging?
+		return V2.ExtractorsService.getExtractorsApiV2ExtractorsGet()
```

Comment:
There is no method called `getExtractorsApiV2ExtractorsGet` exist. Could you double check the backend route. Is there any GET endpoint exist?

Reply:
I think this one went away and what I have is from before. i'll just remove this one since we won't be using it anywhere.